config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
 
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
```

Comment:
- [ ] The routes on lines 7 and 8 repeat some of the routes defined in ones 3 to 5. Kindly remove those last two routes.